webservice/export-catalogs.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var  ensureDb = require('dbjs/valid-dbjs')
+  , memoize = require('memoizee');
+
+module.exports = memoize(function (db) {
+	// ensure db
+	ensureDb(db);
+
+	var catalogs = [];
+	// all enums
+	var enums = db.StringLine.extensions.toArray().filter(function (extension) {
+		return extension.meta && extension.members;
+	}).map(function (extension) {
+		var catalogItem = { name: extension.__id__ };
+		catalogItem.options = extension.members.toArray().map(function (option) {
+			return { name: option, label: extension.meta[option].label };
+		});
+		return catalogItem;
+	});
+	[].push.apply(catalogs, enums);
```

Comment:
@mtuchowski  they all are enums for the end consumer
i don't think this new grouping brings any extra value